subgraphs/rocket-pool/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "license": "MIT",
+  "scripts": {
+    "format": "npx prettier --write ."
+  },
+  "dependencies": {
+    "@graphprotocol/graph-cli": "0.32.0",
```

Comment:
0.37.1: https://www.npmjs.com/package/@graphprotocol/graph-cli

I am not quite sure why we are seeing that error yet: it says it's in this file: `'src/mappings/rocketRewardsPoolMapping.ts'`

Reply:
Actually I was able to repro it locally w the latest version of graph-cli installed globaly